applications/portfolio_manager/src/portfolio_manager/scheduler.py

```
@@ -80,6 +80,39 @@ async def spawn_rebalance_scheduler(
+            positions = alpaca_client.get_open_positions()
+            logger.info(
+                "Account status",
+                cash_amount=account.cash_amount,
+                buying_power=account.buying_power,
+                position_count=len(positions),
+                positions=positions,
+            )
+            await asyncio.sleep(_STATUS_LOG_INTERVAL_SECONDS)
+        except asyncio.CancelledError:
+            logger.info("Status logger cancelled")
+            return
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)